vector/src/main/java/im/vector/app/features/popup/PopupAlertManager.kt

```diff
@@ -218,7 +218,7 @@ class PopupAlertManager @Inject constructor(
         if (!alert.isLight) {
             clearLightStatusBar()
         }
-        val noAnimation = !animate || isAnimationDisabled(activity)
+        val noAnimation = !(animate && activity.isAnimationEnabled())
```

Comment:
I've read it several times, but please double check this change :)

Reply:
took me some time to proceed, but looks correct